pkg/apic/apiserver/models/management/v1alpha1/WatchTopic.go

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is automatically generated
+ */
+
+package v1alpha1
+
+import (
+	"encoding/json"
+
+	apiv1 "github.com/Axway/agent-sdk/pkg/apic/apiserver/models/api/v1"
+)
+
+var (
+	_WatchTopicGVK = apiv1.GroupVersionKind{
+		GroupKind: apiv1.GroupKind{
+			Group: "management",
+			Kind:  "WatchTopic",
+		},
+		APIVersion: "v1alpha1",
+	}
+)
+
+const (
+	WatchTopicScope = ""
```

Comment:
The generator was updated to make this an array of scopes, I do not think it will matter for this resource, however.

Reply:
We will re-generate the source, but you are right it should not matter for WatchTopic resource